cmds/identityd/main.go

```diff
@@ -52,6 +54,13 @@ func main() {
 		log.Fatal().Err(err).Msg("failed to create module root")
 	}
 
+	redis, err := zbus.NewRedisClient(msgBrokerCon)
+	if err != nil {
+		log.Fatal().Err(err).Msg("failed to create zbus client")
+	}
+
+	upgrade := stubs.NewUpgradeModuleStub(redis)
```

Comment:
I really don't like the idea that `identity` depends on upgrade for its operation. Even in zinit units, upgraded indirectly depends on identityd (which is considered a boot service).